app/views/partials/_nav.html.erb

```diff
@@ -11,11 +11,8 @@
         <%= link_to "About", about_path %>
       </div>
       <div class="align-middle table-cell p3">
-        <%= link_to "Register", register_path %>
-      </div>
-      <div class="align-middle table-cell p3">
-        <%= link_to 'Login', login_path, class: "px2"%>
-        <%= link_to 'Sign Out', logout_path, method: :delete %>
+        <%= link_to 'Sign In', '/login', class: "px2"%>
+        <%= link_to 'Sign Out', '/logout', method: :delete %>
```

Comment:
`registration` and `login` links initially planned in the wire framing are replaced by a `sign in` link. Both registration and login were redirected back to the same method of the session controller and were therefore redundant